internal/servicebindingcleanup/service.go

```diff
@@ -0,0 +1,56 @@
+package servicebindingcleanup
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"github.com/kyma-project/kyma-environment-broker/internal"
+	"github.com/kyma-project/kyma-environment-broker/internal/storage"
+)
+
+type BrokerClient interface {
+	Unbind(binding internal.Binding) error
+}
+
+type Service struct {
+	dryRun          bool
+	brokerClient    BrokerClient
+	bindingsStorage storage.Bindings
+}
+
+func NewService(dryRun bool, client BrokerClient, bindingsStorage storage.Bindings) *Service {
+	return &Service{
+		dryRun:          dryRun,
+		brokerClient:    client,
+		bindingsStorage: bindingsStorage,
+	}
+}
+
+func (s *Service) PerformCleanup() error {
+	slog.Info(fmt.Sprintf("Fetching Service Bindings with expires_at <= %q", time.Now().UTC().Truncate(time.Second).String()))
+	bindings, err := s.bindingsStorage.ListExpired()
+	if err != nil {
+		return err
+	}
+
+	slog.Info(fmt.Sprintf("Expired Service Bindings: %d", len(bindings)))
+	if s.dryRun {
```

Comment:
I wonder if:
```go
if !s.dryRun {
   slog.Info()
   ...
}
return nil
```
is easier to read.

Reply:
I agree, even better if we drop the conditional clause for sending requests:
```go
if s.dryRun {
  return nil
}
slog.Info("Requesting Service Bindings removal...")
...
```
What do you think?